File: C++ Code/Basics/Mensuration/3 D/Cuboid.cpp
// WAP to calculate Volume,TSA and LSA of a cuboid  .
#include<iostream>
using namespace std;
int main(){
    float l,b,h;
    cout<<"Enter the length : ";
    cin>>l;
    cout<<"Enter the breadth : ";
    cin>>b;
    cout<<"Enter the height : ";
    cin>>h;
    cout<<"Volume = "<<l*h*b<<endl;
    cout<<"Total Surface Area = "<<2*(l*h+l*b+b*h)<<endl;
    cout<<"Lateral Surface Area = "<<2*(l*h+b*h);
    return 0;
}

File: Java Code/Basics/Mensuration/Cuboid.java
import java.util.*;

public class Cuboid{
  public static void main(String[] args){
    Scanner sc=new Scanner(System.in);
    System.out.println("For the cuboid :");
    System.out.print("Enter length : ");
    float a=sc.nextFloat();
    System.out.print("Enter breadth : ");
    float b=sc.nextFloat();
    System.out.print("Enter height : ");
    float c=sc.nextFloat();
    float TSA=2*(a*b+a*c+b*c);
    float LSA=2*(a*c+b*c);
    float V=a*b*c;
    System.out.println("Lateral Surface Area = "+LSA);
    System.out.println("Total Surface Area : "+TSA);
    System.out.println("Volume : "+V);
  }
}